README.md

```diff
@@ -1 +1,49 @@
-# apple-website-clone
\ No newline at end of file
+# apple-website-clone
+
+# PROJECT: PROJECT: BUILDING WITH BACKGROUNDS AND GRADIENTS (Microverse Collaborative Project)
+
+> This project consists on using of plain HTML and CSS to create a website.
+> A reproduction of "apple" website was used to complete the project (https://web.archive.org/web/20140301004610/http://www.apple.com/)
+
+#### Techniques used for positioning elements : 
+
+flex
+background images
+gradient
+
+![screenshot](./src/webpage_screenshot.png)
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+- [Live Demo Link](https://raw.githack.com/Ahmed-Benj/apple-website-clone/tree/development)
```

Comment:
please fix the live demo link it's not working

Reply:
done